third_party/crypto/acme/acme.go

```diff
@@ -488,15 +488,7 @@ func (c *Client) AcceptChallenge(ctx context.Context, chal *Challenge) (*Challen
 		return nil, err
 	}
 
-	auth, err := keyAuth(c.Key.Public(), chal.Token)
-	if err != nil {
-		return nil, err
-	}
-
-	req := struct {
-		Auth string `json:"keyAuthorization"`
-	}{auth}
-	res, err := c.postWithJWSAccount(ctx, chal.URL, req)
+	res, err := c.postWithJWSAccount(ctx, chal.URL, json.RawMessage(`{}`))
```

Comment:
@bluecmd I'm not quite understanding this change here - how come we no longer send the `keyAuthorization` parameter in this call?

Reply:
Sending a `keyAuthorization` in the challenge response is a legacy artifact from before RFC 8555. Both the client and the server can compute the value independently so there was no reason to include it in the JWS body. 

Pebble 2.0.0+ (or older versions with `-strict=true`) reject challenge POSTs that contain a key auth.